Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6
```

Comment:
That container I linked is a Wave internal one, so you'd need to `make start` to log in.

For a public project `python:3.6.0-alpine` is fine, but you'll need to install the `pg` drivers before you install requirements - add this to your `Dockerfile`:
```
RUN apk add --no-cache --virtual .pg-drivers \
    postgresql-client \
    postgresql-dev
```
You'll have to do similar for MySQL too - [thi](https://pkgs.alpinelinux.org/packages) is the best way to figure out package names, or check out one of our apps that we've converted which use MySQL.